tests/integration/tests/conftest.py

```diff
@@ -25,6 +25,29 @@ def _harness_clean(h: harness.Harness):
         h.cleanup()
 
 
+def _generate_inspection_report(h: harness.Harness, instance_id: str):
+    LOG.debug("Generating inspection report for %s", instance_id)
+
+    inspection_path = Path(config.INSPECTION_REPORTS_DIR)
+    result = h.exec(
```

Comment:
maybe ignore failures while retrieving the inspection reports.

eg for the etcd instances, where k8s-snap is not installed. you could add maybe check=False for that